docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -245,6 +245,29 @@ For flat typed networks, regular kube-proxy may be used to provide pod access to
 For VPC-isolated networking, service IP support has to be provided w/o relying on iptable/IPVS rules on host netns. In other words, traditional kube-proxy can not used in VPC-isolated networking. A dedicated controller may be employed to establish & maintain service IP/pod IP mappings.
 (**TBD: more details required**)
 
+### Endpoints
+Each endpoints object associates with a network, the same as its service.
+
+For the services of _kubernetes_ and _kube-dns_ (in fact they are kubernetes_\{network\} and kube-dns_\{network\}), there are two special set of endpoints objects:
+
+| endpoints of | example | notes |
```

Comment:
This table is pretty difficult to understand.  Maybe just use two bullets to explain there are endpoints for two special services? Or rename the table column header and use specific endpoint/service names to make the table easier to understand.

Reply:
will update it to make it easier to understand.